src/components/Bottombar/index.element.tsx

```diff
@@ -0,0 +1,8 @@
+import styled from "styled-components";
+
+export const BottomBarContainer = styled.div<{ extend: 1 | 0 }>`
+  width: 100%;
+  height: ${(props) => (props.extend === 1 ? "8.8rem" : "3.4rem")};
```

Comment:
```suggestion
  height: ${(props) => (props.extend? "8.8rem" : "3.4rem")};
```
바로 밑 리뷰처럼 boolean 속성을 바로 가져다가 쓰면 좋을 것 같아용